src/index.html

```diff
@@ -17,16 +17,31 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form_big"
+      data-qa="big"
+    >
```

Comment:
The `<form>` tag is self-closing, which is not valid in HTML. Form tags should have a separate closing tag. Change `<form />` to `<form>` and ensure it has a corresponding `</form>`.